### PR TITLE
Bugfix/image upload

### PR DESCRIPTION
**Pull request by Shreets**
Show spinner in the Lab Result and Medication page while the image is being compressed and uploaded 
192761538_523167192145939_6807603724613766750_n

### CODE REVIEW

File: src/pages/ImagesModal.js [hunk 1]
```diff
@@ -40,6 +44,7 @@ const ImagesModal = (props) => {
           setFileAspects([...fileAspects, fileInfo]);
           setMedicationFile([...medicationFile, imageFileInfo]);
         };
+        setIsLoading(false);
         onClose();
       } catch (err) {}
```
Comment (nastikue) [inline src/pages/ImagesModal.js:49]:
```suggestion
        onClose();
      } catch (err) {
        console.error(err);
      } finally {
        setIsLoading(false);
      }
```

File: src/pages/ImagesModal.js [hunk 2]
```diff
@@ -53,33 +58,45 @@ const ImagesModal = (props) => {
...
+              <>
+                <div className="camera-options-wrap display-flex border-radius">
+                  <label className="cameraButton">
+                    Take photo
+                    <input
+                      type="file"
+                      accept="image/*"
+                      capture=""
+                      onChange={imageHandler}
+                      onClick={(e) => (e.target.value = null)}
```
Comment (nastikue) [inline src/pages/ImagesModal.js:79]: strange functionality: I don't think it is a good idea to mutate e.target.value in our function

File: src/pages/ImagesModal.js (hunk 1 above)
Comment (Shreets) [reply src/pages/ImagesModal.js]: resolved

File: src/pages/ImagesModal.js (hunk 2 above)
Comment (Shreets) [reply src/pages/ImagesModal.js]: the onClick functionality has been removed

**Review by nastikue — APPROVED** [review]

**Pull request merged**